yarn.lock

```diff
@@ -2678,6 +2678,10 @@
   version "0.0.0"
   uid ""
 
+"@kbn/mapbox-gl@link:bazel-bin/packages/kbn-mapbox-gl/npm_module":
+  version "0.0.0"
```

Comment:
Why is version 0.0.0 here but 1.0.0 in kbn/mapbox-gl?

Reply:
I don't know. It seems to be pretty consistent, also for other bazel-ackages in the yarn-lock.

cc @mistic do you know why this version is `0.0.0`?